src/sagemaker/xgboost/defaults.py

```diff
@@ -15,14 +15,20 @@
 
 XGBOOST_NAME = "xgboost"
 XGBOOST_1P_VERSIONS = ["1", "latest"]
-XGBOOST_VERSION_0_90 = "0.90"
+
+# TODO: evaluate usefulness of these constants. they are defined here and only used here
```

Comment:
they're used in `XGBOOST_SUPPORTED_VERSIONS` which is used in `get_image_uri` iirc